version_template.py

```diff
@@ -127,12 +127,23 @@ def dirty():
         return False  # In case the git call fails.
 
 
-def expand_(name=None):
+def expand_(name=None, others=None):
+    """
+    Returns a string of all the globals and additional variables passed as the
+    others keyword argument.
+
+    :param str name: If set, only the value of the given symbol is returned.
+    :param dict others: A dictionary of additional variables to be included in
```

Comment:
We've been moving towards just `:param whatever:` here and the type hints in the method name. But we can clean this up if/when we actually MyPy the file, I think.